### ISSUE TITLE
[HacktoberFest] UI Improvements

### ISSUE THREAD

**Issue by vinitshahdeo** [vinitshahdeo/TwitterSentimentAnalysis#1]
Kindly come up with an **eye-catching layout** for the homepage (i.e. `index.php`)

- Feel free to play with **fonts/colors** in CSS files. 
- **Do not make any changes to other files.**
- Make sure that **anything shouldn't break** i.e. (core functionality written in `php`)

> _PS - I don't have any wireframe in my mind if you come up with one, please comment on this thread._

**Comment by misaelgosantos** [comment]
can i do it ?

**Comment by misaelgosantos** [comment]
please see my pull request

**Comment by vinitshahdeo** [comment]
@MisaelSantos Yeah just saw. It's just a small change. Can you please make some more modifications?

**Comment by misaelgosantos** [comment]
> 
> 
> @MisaelSantos Yeah just saw. It's just a small change. Can you please make some more modifications?

I will make

**Comment by misaelgosantos** [comment]
Please checkout my new pull request and give feedback

**Comment by vinitshahdeo** [comment]
@MisaelSantos Thanks for the PR. **You may a few more issues [here](https://github.com/vinitshahdeo/HacktoberFest2K19).**

**Comment by kamwil42** [comment]
Hey,
Is it still open? I'd also love to help improve the UI, I think there are a few changes that could make it look better

**Comment by vinitshahdeo** [comment]
@Pervicorn Yes, please go ahead.

**Comment by kamwil42** [comment]
Alright, I've made a pull request

**Comment by vinitshahdeo** [comment]
## CHECK OUT [THIS LINK](https://corona-cases-india.netlify.com/) FOR LIVE CORONA UPDATES

Hello everyone,

I hope you are staying safe at your home and enjoying the contribution to your awesome open source projects. **I've created a web app to display the latest corona updates, please check [here](https://corona-cases-india.netlify.com/).**

#### :wave: [DO THE FIVE](https://www.mohfw.gov.in/): Help stop coronavirus

1. **HANDS**: Wash them often
2. **ELBOW**: Cough into it
3. **FACE**: Don't touch it
4. **SPACE**: Keep safe distance
5. **HOME**: Stay if you can

#### [COVID-19](https://corona-cases-india.netlify.com/): Stay Home<sup>Stay Safe</sup>

- Avoid Handshakes 🤝 ❌
- Do Namastey 🙏 ✔️


---

<sup>**PS: I know it's off-topic but **Yes**, COVID-19 everyone's talking about. Please stay safe at your home and enjoy contributing to your projects.** </sup>

**Comment by vinitshahdeo** [comment]
### Hello everyone,

> # Do check out my latest [COVID-19 😷Tracker](https://indiafightscorona.netlify.app/) | INDIA 🇮🇳

[![COVID-19 Tracker by Vinit Shahdeo](https://user-images.githubusercontent.com/20594326/79394735-d1817b80-7f95-11ea-90a4-7b99e86f8560.png)](https://indiafightscorona.netlify.app/)

---
<sup><strong> I've just revamped the UI(WIP). You can check the old website [here](https://corona-cases-india.netlify.app/). Consider leaving a star [here](https://github.com/vinitshahdeo/COVID19).  :star: <br>(To be open-sourced pretty soon.)</strong></sup>

[![GitHub stars - COVID-19 Vinit Shahdeo](https://img.shields.io/github/stars/vinitshahdeo/COVID19?label=LEAVE%20A%20Star%20on%20GitHub&logo=github&style=for-the-badge)](https://github.com/vinitshahdeo/COVID19/)

**Comment by 0xbn0xi0us** [comment]
@vinitshahdeo 

hello sir can I work on this issue.

**Comment by 0xbn0xi0us** [comment]
please review my pull request #16 @vinitshahdeo
